pallets/loan/src/benchmarking.rs

```diff
@@ -224,11 +225,13 @@ fn activate_test_loan_with_defaults<T: Config>(
 	<T as LoanConfig>::Rate: From<Rate>,
 	<T as LoanConfig>::Amount: From<Amount>,
 {
-	let loan_type = LoanType::BulletLoan(BulletLoan::new(
+	let loan_type = LoanType::CreditLineWithMaturity(CreditLineWithMaturity::new(
```

Comment:
We should probably bench all three types, take the max, then set up the extrinsics to refund the difference based on the loan type actually used.

I think more complex benchmarking and making sure we don't use more block weight than our actual extrinisc times can be a future thing to work on, though

Reply:
true. Right now i took the one with more computation and storage combined. We have to fine tune these benchmarks down the line. Not a major difference between each types for prioritise this for V1 atleast.